tests/test_formatter.py

```diff
@@ -94,3 +94,31 @@ def check_example(name):
             cython_example.foo()
         except Exception:
             MyFormatter().print_exception()
+
+    with check_example("blank_visible"):
+        try:
+            blank_lines()
+        except Exception:
+            MyFormatter(options=Options(blank_lines=BlankLines.VISIBLE)).print_exception()
+
+    with check_example("blank_single"):
+        try:
+            blank_lines()
+        except Exception:
+            MyFormatter(options=Options(blank_lines=BlankLines.SINGLE)).print_exception()
+
+
+    with check_example("blank_visible_no_linenos"):
+        try:
+            blank_lines()
+        except Exception:
+            MyFormatter(show_linenos=False, options=Options(blank_lines=BlankLines.VISIBLE)).print_exception()
+
+
+def test_invalid_single_option():
+    try:
+        MyFormatter(show_linenos=False, options=Options(blank_lines=BlankLines.SINGLE))
+    except ValueError:
+        assert True
+    else:
+        assert False
```

Comment:
```suggestion
    with pytest.raises(ValueError):
        MyFormatter(show_linenos=False, options=Options(blank_lines=BlankLines.SINGLE))
```

and an `import pytest` at the top.

Reply:
Will do.